Lint issue with new version of flake8-comprehensions plugin

[flake8-comprehensions](https://pypi.org/project/flake8-comprehensions/) have released a new version (3.2.1), and now when running ```./ci/code_checks.sh lint``` I get this output:

```
flake8 --version
3.7.9 (flake8-comprehensions: 3.2.1, mccabe: 0.6.1, pycodestyle: 2.5.0, pyflakes: 2.1.1) CPython 3.8.1 on Linux
Linting .py code
./pandas/tests/io/parser/test_network.py:208:21: C412 Unnecessary set comprehension - 'in' can take a generator.
Linting .py code DONE
```

(feel free to close this issue if this is happening because I have python version 3.8.1)

<details>

INSTALLED VERSIONS
------------------
commit           : 7c94949dc89c62cae1bc647acd87266d6c3a0468
python           : 3.8.1.final.0
python-bits      : 64
OS               : Linux
OS-release       : 5.4.13.a-1-hardened
machine          : x86_64
processor        : 
byteorder        : little
LC_ALL           : None
LANG             : en_US.UTF-8
LOCALE           : en_US.UTF-8

pandas           : 1.0.0rc0+107.g7c94949dc
numpy            : 1.18.1
pytz             : 2019.3
dateutil         : 2.8.1
pip              : 19.3.1
setuptools       : 45.1.0
Cython           : 0.29.14
pytest           : 5.3.3
hypothesis       : 5.2.0
sphinx           : 2.3.1
blosc            : 1.8.3
feather          : None
xlsxwriter       : 1.2.7
lxml.etree       : 4.4.2
html5lib         : 1.0.1
pymysql          : None
psycopg2         : None
jinja2           : 2.10.3
IPython          : 7.11.1
pandas_datareader: None
bs4              : 4.8.2
bottleneck       : 1.3.1
fastparquet      : 0.3.2
gcsfs            : None
lxml.etree       : 4.4.2
matplotlib       : 3.1.2
numexpr          : 2.7.1
odfpy            : None
openpyxl         : 3.0.1
pandas_gbq       : None
pyarrow          : None
pytables         : None
pytest           : 5.3.3
s3fs             : 0.4.0
scipy            : 1.4.1
sqlalchemy       : 1.3.12
tables           : 3.6.1
tabulate         : 0.8.6
xarray           : 0.14.1
xlrd             : 1.2.0
xlwt             : 1.3.0
xlsxwriter       : 1.2.7
numba            : 0.47.0
</details>

--- a/pandas/tests/io/parser/test_network.py
+++ b/pandas/tests/io/parser/test_network.py
@@ -205,7 +205,7 @@ class TestS3:
         with caplog.at_level(logging.DEBUG, logger="s3fs"):
             read_csv("s3://pandas-test/large-file.csv", nrows=5)
             # log of fetch_range (start, stop)
-            assert (0, 5505024) in {x.args[-2:] for x in caplog.records}
+            assert (0, 5505024) in (x.args[-2:] for x in caplog.records)
 
     def test_read_s3_with_hash_in_key(self, tips_df):
         # GH 25945

